clang/include/clang/Frontend/FrontendActions.h

```diff
@@ -299,6 +299,14 @@ class PrintPreprocessedAction : public PreprocessorFrontendAction {
   bool hasPCHSupport() const override { return true; }
 };
 
+class GetDependiciesByModuleNameAction : public PreprocessOnlyAction {
```

Comment:
Typo. `GetDependiciesByModuleNameAction` -> `GetDependenciesByModuleNameAction`.

Reply:
I think inheriting from `PreprocessOnlyAction` here is fine. 👍 We don't need to load any PCH files.